source/tutorials/gneiss.rst

```diff
@@ -0,0 +1,184 @@
+Differential abundance analysis with gneiss
+===========================================
+
+.. note:: This guide assumes you have installed QIIME 2 using one of the procedures in the :doc:`install documents <../install/index>` and completed the :doc:`moving pictures tutorial <moving-pictures>`.
+
+
+In this tutorial you will learn how to perform differential abundance analysis using balances in gneiss.  The main problem that we will focus on is how to identify differentially abundant taxa in a compositionally coherent way.
+
+Compositionality refers to the issue of dealing with proportions.  Since the sequencing depth is not typically biologically relevant, the microbial abundances are best interpreted as proportions.  Because of this, it becomes challenging to infer exactly which microbes are changing -- since proportions add to one, the change of a single microbe will also change the proportions of the remaining microbes.
+
+Consider the following example:
+
+.. image:: images/gneiss-bars.jpg
+
+On the left, we have the true abundances of ten species, and the first species doubles between Time point 1 and Time point 2.  When we normalize these to proportions, it appears as if all of the species have changed between the two time points.  Looking at proportions alone, we would never realize this problem, and we actually cannot exactly determine which species are changing based on proportions alone.
+
+While we cannot exactly solve the problem of identifying differentially abundant species, we can relax this problem and ask which partitions of microbes are changing.  In the case above, if we compute the ratio between the first species and the second species, that ratio will be 1:1 at Time point 1, and 2:1 at Time point 2 for both the original abundances and the proportions.  This is the type of question that balances try to solve.
+Rather than focusing on individual taxa, we can focus on the log ratios, since the log ratios are consist between the original abundances and the observed proportions of the species are observed.  The concept of calculating balances for multiple species can be extended to trees as shown in the following example.
+
+.. image:: images/gneiss-balances.jpg
+
+On the right, we define a tree, where each of the tips corresponds to a taxon and their corresponding proportions.  The internal nodes define the log ratio between the taxa underneath.  On the left, we have the same tree, but one of the taxa abundances changes.  As we have observed before, the proportions of all of the taxa will change, but looking at the balances, only the balances containing the purple taxa will change.  In this case, balance :math:`b_3` won't change, since it only considers the ratio between the red and green taxa.  By looking at balances instead proportions, we can control for the variance by restricting observations to only focus on the taxa within the balance.
```

Comment:
Can we flip the order that we discuss the images? It feels strange talking about the image on the right, then the image on the left.

Reply:
Turns they were discussed left-to-right, they just had the wrong side listed.